modules/collect/src/main/java/com/opengamma/strata/collect/result/Result.java

```diff
@@ -594,6 +609,19 @@ public boolean isFailure() {
     return failure != null;
   }
 
+  /**
+   * Executes the given consumer if the result represents a failure.
+   *
+   * This is the opposite of {@link #ifSuccess(Consumer)}.
```

Comment:
No need for this - it is self explanatory. And if it were here, it would need `<p>` above it.